### ISSUE TITLE
Plug-in fails with cryptic error on non-JDK 1.8 machines.

### ISSUE THREAD

**Issue by tonycosentini** [uber-common/infer-plugin#13]
Running the Infer plug-in on a machine with JDK 7 - I get this error:
```
Caused by: com.google.common.util.concurrent.ExecutionError: java.lang.UnsupportedClassVersionError: com/uber/infer/InferAndroidPlugin : Unsupported major.minor version 52.0
```

I think we need to make sure we explicitly support pre-JDK 1.8 - should be an easy fix.

**Comment by spirosoik** [comment]
+1 exactly the same problem.

**Comment by jaredsburrows** [comment]
@tonycosentini @clonetwin26 What was this solution to this?

**Comment by clonetwin26** [comment]
Should be fixed and included in the 0.7.3 release
https://github.com/uber-common/infer-plugin/pull/21
